src/register/register.ts

```diff
@@ -146,10 +316,49 @@ export class Register {
         })
       );
 
-      Register.registers[register].text = text;
-    }
+      // Harmonize newline character
+      text = text.replace(/\r\n/g, '\n');
```

Comment:
So when i think about it the "The default end of line character" (comment for the setting) is not the problem, i would need the "current system clipboard end of line character" because i had to romve the `\r` only on windows because vscode adds it when `\n` is inserted. And in case of unix system replacing `\n` with `\n` is unnecessary.